### PR TITLE
wrapping create stats query

### PR DESCRIPTION
**Pull request by shimonewman**

### CODE REVIEW

File: src/Appwrite/Usage/Calculators/TimeSeries.php [hunk 1]
```diff
@@ -422,36 +423,39 @@ public function __construct(string $region, Database $database, InfluxDatabase $
      */
     private function createOrUpdateMetric(string $projectId, string $time, string $period, string $metric, int $value, int $type): void
     {
+
```
Comment (abnegate) [inline src/Appwrite/Usage/Calculators/TimeSeries.php:426]: Let's remove the blank line

File: src/Appwrite/Usage/Calculators/TimeSeries.php [hunk 2]
```diff
@@ -422,36 +423,39 @@ public function __construct(string $region, Database $database, InfluxDatabase $
...
+                $document = $database->getDocument('stats', $id);
+                if ($document->isEmpty()) {
+                    $database->createDocument('stats', new Document([
                     '$id' => $id,
                     'period' => $period,
                     'time' => $time,
                     'metric' => $metric,
                     'value' => $value,
                     'type' => $type,
                     'region' => $this->region,
```
Comment (abnegate) [inline src/Appwrite/Usage/Calculators/TimeSeries.php:442]: Let's fix indentation here

Comment (shimonewman) [reply src/Appwrite/Usage/Calculators/TimeSeries.php:441]: That's a git display thing.
If you look at the file via idea, it is okay.

**Review by christyjacob4 — APPROVED** [review]

**Review by lohanidamodar — COMMENTED** [review]
I believe apart from the auth skip others are just formatting changes. Looks good.

**Pull request merged**